pkg/apis/pingcap/v1alpha1/tidbcluster_component.go

```diff
@@ -59,6 +61,10 @@ type componentAccessorImpl struct {
 	ComponentSpec *ComponentSpec
 }
 
+func (a *componentAccessorImpl) StatefulSetUpdateStrategy() apps.StatefulSetUpdateStrategyType {
+	return a.ComponentSpec.StatefulSetUpdateStrategy
```

Comment:
Should we also add a field in the `spec` just as `imagePullPolicy` so that we can control this behavior of all components with only one field?